misaligned.cpp

```diff
@@ -1,21 +1,39 @@
 #include <iostream>
 #include <assert.h>
 
+void checkPairNumberValidity(int pair_no)
+{
+    assert(pair_no>=1 && pair_no<=25);
+}
+
+void displayPairNumber(int x, int y)
+{
+    int pair_number = x * 5 + y;
+    std::cout<<pair_number+1;   //Pair numbers should start from 1
+    checkPairNumberValidity(pair_number);

```

Comment:
Please add test cases to describe or express the behavior for the existing code. Do not fix the bug implementation code or add new code. Refactoring is ok i.e. segregating into different functions.